pages/documentation/migration-V3.md

```diff
@@ -21,8 +21,8 @@ subnav:
   href: '#4-integrate-any-recent-uswds-changes'
 - text: 5. Update to Sass module syntax
   href: '#5-update-to-sass-module-syntax'
-- text: 6. Optimize your installation with component packages
-  href: '#6-optimize-your-installation-with-component-packages'
+- text: 6. Optimize your installation
+  href: '#6-optimize-your-installation'
```

Comment:
I renamed this section because its content seemed to include more than just component packages.

Reply:
Nice one!